app/view_utils/paypal_helper.rb

```diff
@@ -85,7 +85,19 @@ def paypal_provisioned?(community_id)
   def open_listings_with_missing_payment_info?(user_id, community_id)
     paypal_active?(community_id) &&
     !user_and_community_ready_for_payments?(user_id, community_id) &&
-    !MarketplaceService::Listing::Query.open_listings_with_price_for(community_id, user_id).empty?
+    open_listings_with_price_and_payment_process?(community_id, user_id)
+  end
+
+  def open_listings_with_price_and_payment_process?(community_id, user_id)
+    listings = MarketplaceService::Listing::Query.open_listings_with_price_for(community_id, user_id)
```

Comment:
Get rid of the price check, it's old logic. Instead get all unique process ids for users listings in this community and iterate through processes to see if any of them requires payments.